src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -11,6 +11,7 @@
 using System.Web;
 using Microsoft.OpenApi.Models;
 using Microsoft.PowerFx.Connectors.Execution;
+using Microsoft.PowerFx.Interpreter;
```

Comment:
Why do we need the new using here and in ServiceFunction.cs?

Reply:
Oh yes! This was something I ended up undoing but I didn't remove the usings. I will go fix those.

Reply:
Done